wolox-books/src/app/pipes/filter-books.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'filterBooks'
+})
+export class FilterBooksPipe implements PipeTransform {
+
+  transform(books: any[], filterText: string): unknown {
```

Comment:
Why the `unknown`? Try to use the correct types. In this case I think it should be `Books[]`

Reply:
I didn't change the unkown from the `ng generate pipe` template, you're right, my bad 😅